content/docs/rfcs/35/README.md

```diff
@@ -0,0 +1,230 @@
+---
+slug: 35
+title: 35/WAKU2-NOISE
+name: Noise Protocols for Waku2 Payload Encryption
+status: raw
+editor: Giuseppe <giuseppe@status.im>
+contributors: 
+---
+
+This specification describes how Waku2 messages can be encrypted
+in order to achieve confidentiality, authenticity, and integrity 
+as well as some form of identity-hiding on communicating parties.
+
+Specifically, it describes how encryption keys can be exchanged using the [Noise protocol Framework](http://www.noiseprotocol.org/noise.html) 
+to allow authenticated encryption/decryption in [10/WAKU2](/spec/10) with [14/WAKU-MESSAGE version 2](/spec/14/#version2). 
+This ultimately allows Waku2 users to instantiate end-to-end encrypted communication channels.
+
+
+This specification is an addition to [26/WAKU-PAYLOAD](/spec/26).
+
+## Design requirements
+
+- *Confidentiality*: 
+The adversary should not be able to learn what data is being sent from one Waku2 endpoint to one or several other Waku2 endpoints. 
+In particular, forward secrecy should be provided on exchanged data. 
```

Comment:
would it make sense to pull out forward secrecy as a separate point?

Reply:
Addressed in https://github.com/vacp2p/rfc/pull/496/commits/8becd7bf7c5bf8575877b8cdc4444a4b9e5d1463